content/platforms/kubernetes/concepts/manage_REC_credentials.md

```diff
@@ -0,0 +1,119 @@
+---
+Title: Manage Redis Enterprise cluster (REC) credentials
+description: 
```

Comment:
```suggestion
linkTitle: Manage cluster credentials
description: 
```

Reply:
I have to be careful about using cluster without specifying the type of cluster. So I think "Manage REC cluster credentials bring more clarity.

Reply:
The existing title will force a line break, which is the main reason to use use linkTitle here.  (Title is the default for the TOC when linkTitle is absent.)